scylla-cdc-replicator/Cargo.toml

```diff
@@ -14,4 +14,5 @@ async-trait = "0.1.51"
 anyhow = "1.0.48"
 itertools = "0.10.3"
 uuid = {version = "1.0.0", features = ["v1"]}
-futures-util = "0.3.19"
\ No newline at end of file
+futures-util = "0.3.19"
+log = "0.4.17"
```

Comment:
I'd recommend using the [`tracing`](https://docs.rs/tracing/latest/tracing/) crate instead. It integrates with asynchronous code better, and the rust driver uses it, too.